lib/core/utils/index.js

```diff
@@ -31,6 +31,7 @@ export { default as getFriendlyUriEnd } from './get-friendly-uri-end';
 export { default as getNodeAttributes } from './get-node-attributes';
 export { default as getNodeFromTree } from './get-node-from-tree';
 export { default as getRootNode } from './get-root-node';
+export { default as getRule } from './get-rule';
```

Comment:
Why are you putting this on `.utils` instead of on `axe`? Not objecting, to it, but would like to know the reason for it.